hrjobcontract/api/v3/HRJobContract.php

```diff
@@ -110,6 +119,35 @@ function civicrm_api3_h_r_job_contract_getlengthofservice($params) {
   return civicrm_api3_create_success($result, $params);
 }
 
+/**
+ * HRJobContract.getlengthofserviceymd
+ * 
+ * Return an object containing years, months and days of Length of Service
+ * for specific 'contact_id'.
+ * Useful for front-end part of Contact Summary block.
+ * 
+ * @param array $params
+ * @return array API result descriptor
+ * @throws API_Exception
+ */
+function civicrm_api3_h_r_job_contract_getlengthofserviceymd($params) {
```

Comment:
I though it may be a good thing to have one entry point for getting these days/months/years in case we want to use it somewhere else (maybe in SSP My Details or so) and not recalculating it in each place using different angular/JS/template methods.